Assets/Scripts/Buddy/BuddyStats.cs

```diff
@@ -4,11 +4,14 @@
 public class BuddyStats : MonoBehaviour
 {
 	public Stat statType = Stat.Invalid;
-	[SerializeField] int startingApples = 10;
-	[SerializeField] int apples = 0;
+	[SerializeField] bool canDecreaseStat = true;
+	[SerializeField] int startingResources = 10;
+	[ReadOnly("Current Resources")]
+	[SerializeField] int resources = 0;
```

Comment:
I think David and I agreed at  some point that serialized fields should also get the underscore since they're technically private.